packages/main/src/backend/export/outputVendors/ynab/ynab.ts

```diff
@@ -196,7 +204,8 @@ export function isSameTransaction(
     Math.abs(transactionToCreate.amount - transactionFromYnab.amount) < 1000 &&
     // In a transfer transaction the payee name changes, but we still consider this the same transaction
```

Comment:
I see someone already tried to support your case by the comment:

> In a transfer transaction the payee name changes, but we still consider this the same transaction

Reply:
It's a different case, my condition is not referring a transfer transaction